### PR TITLE
Stable Release 1.0 --> Switched to Beta Version 0.2

### PR DESCRIPTION
**Pull request by rayapetkova**

### CODE REVIEW

**Review by karastoyanov — APPROVED** [review]

**Review by karastoyanov — APPROVED** [review]

File: db_handle/manage_portfolio.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    """Create ranomly generated category ID"""
+    category_id = []
+    for i in range(1, 7):
+        category_id.append(random.choice(string.digits))
+    category_id = 'CAT-' + ''.join(category_id)
+
+    """Enter the category name"""
+    category_name = input("Enter category name: ")
+
+    # """Calculates the total sub-categories / for new categories is 0 by default"""
```
Comment (karastoyanov) [inline db_handle/manage_portfolio.py]: @rayapetkova We won't use this section with the comments. Could you remove this block of code from the function.

File: db_handle/manage_portfolio.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    """Create ranomly generated sub-category ID"""
+    subcategory_id = []
+    for i in range(1, 7):
+        subcategory_id.append(random.choice(string.digits))
+    subcategory_id = 'SUBCAT-' + ''.join(subcategory_id)
+    
+    """Enter the sub-category name"""
+    subcategory_name = input("Enter the new subcategory name: ")
+
+    # """Calculates the total products inside the sub-category / for new sub-categories is 0 by default"""
```
Comment (karastoyanov) [inline db_handle/manage_portfolio.py]: @rayapetkova We won't use this section with the comments. Could you remove this block of code from the function.

**Review by karastoyanov — CHANGES_REQUESTED** [review]
Check the comments and take actions.

File: db_handle/manage_portfolio.py (hunk 2 above)
Comment (rayapetkova) [reply db_handle/manage_portfolio.py]: Should I remove only row 42 or remove all four - 39, 40, 41 and 42?

Comment (karastoyanov) [reply db_handle/manage_portfolio.py]: Rows from 42 to 44, only the currently commented lines. Same for above function `create_new_category`

**Comment by rayapetkova** [comment]
Last changes: d5526c6
- Removed rows 19-26 and 41-44 inclusive

**Review by karastoyanov — APPROVED** [review]

**Review by karastoyanov — APPROVED** [review]

**Review by karastoyanov — APPROVED** [review]

**Review by karastoyanov — APPROVED** [review]

**Review by karastoyanov — APPROVED** [review]

**Review by karastoyanov — APPROVED** [review]

File: pyqt5-gui/main_menu.py [hunk 3]
```diff
@@ -81,11 +81,13 @@ def __init__(self):
         log_out_button.setFixedWidth(100)
         log_out_button.setFixedHeight(23)
 
+        top_buttons_layout.addStretch(0)
+        top_buttons_layout.addSpacing(1000)
+
         top_buttons_layout.addWidget(favourites_button)
```
Comment (karastoyanov) [inline pyqt5-gui/main_menu.py:131]: sdsad

**Review by karastoyanov — APPROVED** [review]

File: pyqt5-gui/products.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    for col in range(3):
+        current_groupbox = QGroupBox()
+        current_groupbox.setMaximumWidth(400)
+        current_groupbox.setMaximumHeight(500)
+
+        current_vertical_layout = QVBoxLayout()
+
+        product_name = products_names.popleft()
+
+        product_image = QLabel()
```
Comment (karastoyanov) [inline pyqt5-gui/products.py:54]: jghfdjgdf

**Review by karastoyanov — APPROVED** [review]

**Comment by karastoyanov** [comment]
Closing this PR due to missing the deadline. We will keep this beta in the main branch and...hopefully next year we will be able to finish some of the last planned features. 

I will prepare short changelog and release in next few days.

**Review by karastoyanov — APPROVED** [review]

**Pull request merged**